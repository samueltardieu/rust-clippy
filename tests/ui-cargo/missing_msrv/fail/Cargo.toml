[package]
name = "missing_msrv"
version = "0.1.0"
edition = "2024"

[dependencies]
clippy-utilities = "0.2.0"
