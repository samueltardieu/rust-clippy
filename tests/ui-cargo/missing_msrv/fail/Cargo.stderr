error: missing `rust-version` field
  |
  = note: `-D clippy::missing-msrv` implied by `-D warnings`
  = help: to override `-D warnings` add `#[allow(clippy::missing_msrv)]`

error: could not compile `missing_msrv` (bin "missing_msrv") due to 1 previous error
